fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/load/ChunkWriter.java

```diff
@@ -182,19 +183,20 @@ public void writeItems(List<java.lang.Object> arg0) throws Exception {
                     List<Resource> fhirResourceList = (List<Resource>) objResJsonList;
 
                     for (Resource fhirResource : fhirResourceList) {
-                        if (this.resourceType != fhirResource.getClass().getSimpleName()) {
-                            failValidationIds.add(fhirResource.getClass().getSimpleName() + "/" + fhirResource.getId());
+                        String assertedResourceType = fhirResource.getClass().getSimpleName();
+                        if (this.resourceType.equals(assertedResourceType)) {
```

Comment:
is this missing a `!`?
```suggestion
                        if (!this.resourceType.equals(assertedResourceType)) {
```